app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t(".welcome")
+      redirect_to @user
+    else
+      render "new"
+    end
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
```suggestion
  def show; end
```
Em đưa logic find user ra method load_user và dùng trong before_action 
Nhớ xử lý cho trường hợp không tìm thấy user và dùng find_by để tìm